pydelhi.py

```diff
@@ -172,10 +175,10 @@ def github(bot, update):
     chatAction(bot, update, 'http://github.com/pydelhi')
 
 
-def invitelink(bot, update):
-    chatAction(bot, update, '''To prevent spamming we have removed invite
-    link from the group, please ping any one of the admin/moderators of
-    PyDelhi to help you add your friend to the group.''')
+# def invitelink(bot, update):
```

Comment:
I think we need this but with some modification in the text like `Please ping any one of the admin/moderators of PyDelhi to help you add your friend to the group.`

Reply:
Agreed.